### ISSUE TITLE
Add curve types

### ISSUE THREAD

**Issue by stevage** [opengeospatial/wkt#1]
I'm still not fully across how the standards sit together, but it seems there were several different kinds of curve types defined in the original standard.

The circular curves (CIRCULARSTRING, COMPOUNDCURVE, CURVEPOLYGON etc) were at least named (and given WKB type identifiers). They were then fully defined (I think) in the [SQL/MM standard](https://www.iso.org/standard/60343.html).

For context: https://gis.stackexchange.com/questions/407198/where-are-the-wkt-types-circularstring-compoundcurve-etc-actually-defined

I'm not sure the best way to refer to extensions etc in this document, but it would be amiss not to at least note their existence with a few links.

**Comment by cholmes** [comment]
Yeah, I need to dig deeper into the standards to help figure it out, it's
less straightforward than I was hoping. But agree we should at the very
least link to every relevant bit, and try to get each bit extracted and
accessible. Projections I think are another topic.

**Comment by cholmes** [comment]
Ok, I'm digging deeper and coming with more questions than answers, but want to record findings thus far.

Looks like best documentation of SQL/MM curve WKT is https://github.com/postgis/postgis/blob/stable-3.1/doc/bnf-wkt.txt

I'll try to get my hands on the (non-open) standard to see what it says, if it actually defines a WKT.

My latest thought is to try to standardize 'ewkt' - adopt what PostGIS does and make it more formal: https://twitter.com/opencholmes/status/1422246178784092163

My current thinking is we probably should have a 'wkt' repo that is just a more accessible version of the core. And then work on a 'ewkt' spec that extends the core.

**Comment by cleder** [comment]
There is also an [EBNF version](https://github.com/cleder/parsewkt/blob/master/parsewkt/Wkt.ebnf), a translation of the [PostGIS BNF](https://github.com/postgis/postgis/blob/stable-3.1/doc/bnf-wkt.txt)
